Example/AppDelegate.swift

```diff
@@ -16,7 +16,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         if isRunningTests() {
             window!.rootViewController = UIViewController()
         }
-        #if false
+        #if DEBUG
```

Comment:
This code was taken out so that the example applications would build on their own. Can we leave it disabled for now, or is there some other way to enable the UI tests without making the example applications dependent on the testing targets (and all their dependency libraries)?

Reply:
`Example` App does build on it's own. It is not dependent on any testing target. In fact, it's reverse: `MapboxNavigation` test target needs to build `Example` App and run it with special testing command line argument.

UI test is needed due to feature of Maps functionality which exposes it's SKU token only when requests for tiles or anything else is made. At this point, I don't see any other way to check SKU tokens other than building a special UI test App target.